packages/matchbox/src/styles/helpers.js

```diff
@@ -0,0 +1,6 @@
+export const buttonReset = `
```

Comment:
👍 Nice.

Reply:
You _might_ be missing a few things? Especially for mobile devices:
https://gist.github.com/MoOx/9137295

I am trying to get Xcode installed so I can at least use iOS simulator to test things out. No means of using Android for this sort of thing sadly.